### PR TITLE
Fix trig functions

### PR DESCRIPTION
**Pull request by kurtfchua**
added more test cases for trig functions

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@kurtfchua Is there a reason you submitted a new PR rather than pushing the changes against your existing PR. Which would you prefer we review?

File: appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm [hunk 1]
```diff
@@ -1789,10 +1789,12 @@
           360))
 
 (define (sin-degrees degrees)
-  (sin (degrees->radians-internal degrees)))
+  (if =0 (modulo degrees 180) 0
```
Comment (ewpatton) [inline appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm]: This should be `  (if (= 0 (modulo decrees 180) 0`

File: appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm [hunk 2]
```diff
@@ -1789,10 +1789,12 @@
...
+    (sin (degrees->radians-internal degrees)))
 
 (define (cos-degrees degrees)
-  (cos (degrees->radians-internal degrees)))
+  (if =90 (modulo degreees 180) 0
```
Comment (ewpatton) [inline appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm]: This line suffers the same syntactical issue as above.

**Pull request closed without merging**